assets/app/scripts/directives/resources.js

```diff
@@ -45,9 +53,27 @@ angular.module('openshiftConsole')
     return {
       restrict: 'E',
       scope: {
-        pods: '='
+        pods: '=',
+        projectName: '@?' //TODO optional for now
       },
-      templateUrl: 'views/_pods.html'
+      templateUrl: 'views/_pods.html',
+      controller: function($scope) {
+        $scope.phases = [
+          "Failed",
+          "Pending",
+          "Running",
+          "Succeeded"
```

Comment:
I'd vote for the warning style. Right now it's a general, you-should-really-look-at-this callout. This seems to fall in that category.

I'd really like some way to tell users what triggered the warning, though.